src/PhpWord/Shared/Html.php

```diff
@@ -959,32 +959,32 @@ protected static function parseLink($node, $element, &$styles)
     }
 
     /**
-    * Render horizontal rule
-    * Note: Word rule is not the same as HTML's <hr> since it does not support width and thus neither alignment
-    *
-    * @param \DOMNode $node
-    * @param \PhpOffice\PhpWord\Element\AbstractContainer $element
-    */
+     * Render horizontal rule
+     * Note: Word rule is not the same as HTML's <hr> since it does not support width and thus neither alignment
+     *
+     * @param \DOMNode $node
+     * @param \PhpOffice\PhpWord\Element\AbstractContainer $element
+     */
     protected static function parseHorizRule($node, $element)
     {
         $styles = self::parseInlineStyle($node);
 
         // <hr> is implemented as an empty paragraph - extending 100% inside the section
         // Some properties may be controlled, e.g. <hr style="border-bottom: 3px #DDDDDD solid; margin-bottom: 0;">
 
-        $fontStyle = $styles + ['size' => 3];
+        $fontStyle = $styles + array('size' => 3);
```

Comment:
why you changed [] => array() ???

Reply:
https://github.com/PHPOffice/PHPWord/blob/develop/src/PhpWord/Shared/Html.php   
To keep the same code style. Other code of 'Html.php' file use 'array()' instead of '[]' too.